assets/src/scripts/monitoring-location/components/hydrograph/index.js

```diff
@@ -64,38 +64,41 @@ export const attachToNode = function(store,
         return;
     }
 
-    showDataLoadingIndicator(true);
+    showDataIndicators(true);
 
     const initialPeriod = startDT && endDT ? 'custom' : period || 'P7D';
     const initialStartTime = startDT ?
         DateTime.fromISO(startDT, {zone: config.locationTimeZone}).toISO() : null;
     const initialEndTime = endDT ?
         DateTime.fromISO(endDT, {zone: config.locationTimeZone}).endOf('day').toISO() : null;
+    const initialLoadCompare = compare === 'true' || compare === true ? true : false;
```

Comment:
since the parameters passed in are strings if they come from the URLSearchParams, this translates the string to a boolean.